app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/MocksTable.tsx

```diff
@@ -1,28 +1,28 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useMemo } from "react";
```

Comment:
Missing enhancing records logic

- if records contains mock within collection -> show collection also
- if records contains collection -> show all the mocks within it

https://github.com/requestly/requestly/blob/23c98f37a4df9f836562a2cebaab6899e007dde6/app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/utils/rules.ts#L65-L91